docs/Probe_Calibrate.md

```diff
@@ -0,0 +1,65 @@
+This document describes the method for calibrating the x, y, and z
+offsets of an "automatic z probe" in Klipper. This is useful for users
+that have a `[probe]` or `[bltouch]` section in their config file.
+
+# Calibrating probe X and Y offsets
+
+To calibrate the X and Y offset, navigate to the OctoPrint "Control"
```

Comment:
should it be issue G1 commands rather than or in addition to octoprint control?

Reply:
Well, the high level idea is to ask the user to get the nozzle near the center of the bed.  If you think there's a better way to word that then let me know.